components/automate-ui/src/app/entities/automate-settings/automate-settings.requests.ts

```diff
@@ -3,15 +3,20 @@ import { HttpClient } from '@angular/common/http';
 import { Observable } from 'rxjs';
 import { map } from 'rxjs/operators';
 
+
 import {
   JobSchedulerStatus,
   RespJob,
   IngestJob,
-  IngestJobs,
-  RespJobSchedulerStatus
+  RespJobSchedulerStatus,
+  UnfurledJob,
+  JobRequestBody,
+  InfraJobName,
+  JobCategories
 } from './automate-settings.model';
 
 import { environment } from '../../../environments/environment';
```

Comment:
Prefer to avoid such "parented" paths.
```suggestion
import { environment } from 'environments/environment';
```